src/tests/ftest/server/metadata.py

```diff
@@ -280,6 +280,10 @@ def test_metadata_fillup(self):
             "Phase 3: passed (created %d / %d containers)", len(self.container), loop)
         self.log.info("Test passed")
 
+        # instruct tearDown to skip containers destroy
+        # (we want to invoke pool destroy with a full metadata rdb)
+        self.skip_destroy_containers = True
```

Comment:
```suggestion
        try:
            self.destroy_pools(self.pool)
            self.log.info("Pool with a full metadata was successfully destroyed.")
        except TestFail as error:
            self.log.error("Pool destroy with a full metadata failed!")
        self.container = None
```
Another option, if you don't want to touch test.py, is to manually destroy the pool here and set None to self.container so that the container destroy step in the tearDown will be skipped.

Reply:
@phender Probably has an opinion